libs/stats/docs/pq-cfg.yaml

```diff
@@ -0,0 +1,69 @@
+# Sample configuration file for `odc-stats run` command
+#
+#    odc-stats run --config <this-file|content of this file> ...
+
+# Configure Plugin and Product Attributes
+plugin: pq
+
+# Generic product attributes
+product:
+  name: pc_s2_annual
+  short_name: pc_s2_annual
+  version: 0.0.1
+  product_family: pixel_quality_statistics
+
+  # specific to a site: Africa
+  # (for AU just keep defaults)
+  #collections_site: collections.digitalearthafrica.org
+  collections_site: explorer.digitalearth.africa
+  producer: digitalearthafrica.org
+
+  # Any extra fields to add to .properties section of the STAC/EO3
+  # for each dataset document
+  properties: {}
+
+# Plugin Specific Configuration
+# defaults for PQ are currently this
+#
+# plugin_config:
+#   filters: [[2, 5], [0, 5]]
+#   resampling: nearest
+
+# Other common settings
```

Comment:
delete things that were documentation sake.

Reply:
Done.

Reply:
Done.

Reply:
Done.